src/main/java/br/unb/cic/analysis/ioa/InterproceduralOverrideAssignment.java

```diff
@@ -28,12 +28,15 @@
     private FlowSet<DataFlowAbstraction> res;
     private Body body;
 
-    public InterproceduralOverrideAssignment(AbstractMergeConflictDefinition definition) {
+    private Logger logger;
```

Comment:
logger é apenas para ajudar no debug? É necessário manter isto?

Reply:
Eu ainda tenho algumas coisas para implementar. Então vou continuar usando. Mas se acharem necessário, posso remover e adicionar novamente quando for continuar o desenvolvimento. O que acha @pauloborba e @galilasmb?

Reply:
acho que pode deixar sem problema